routes/gamesRouter.js

```diff
@@ -1,163 +1,176 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const Game = require('../models/schemas/games');
+const Game = require("../models/schemas/games");
 
-router.get('/', (req, res) => {
-    Game.find({}, (err, games) => {
-        if (err) {
-            console.log(err);
-        } else {
-            res.status(200).json(games);
-        }
-    })
+router.get("/", (req, res) => {
+  Game.find({}, (err, games) => {
+    if (err) {
+      console.log(err);
+    } else {
+      res.status(200).json(games);
+    }
+  });
 });
 
-router.post('/add', (req, res) => {
-    const title = req.body.title;
-    const category = req.body.category;
-    const ageRating = req.body.ageRating;
-    const violence = req.body.violence;
-    const sexAndNudity = req.body.sexAndNudity;
-    const alcoholAndDrugs = req.body.alcoholAndDrugs;
-    const gambling = req.body.gambling;
-    const explicitLanguage = req.body.explicitLanguage;
-    const averageScore = req.body.averageScore;
-    const publisher = req.body.publisher;
-    const releaseDate = req.body.releaseDate;
-    new Game({ title, 
-        category, 
-        ageRating, 
-        violence, 
-        sexAndNudity, 
-        alcoholAndDrugs, 
-        gambling, 
-        explicitLanguage,
-        averageScore,
-        publisher,
-        releaseDate}).save((err, result) => {
-        res.status(200).send({ status: result });
-    });
+router.post("/add", (req, res) => {
+  const {
+    title,
+    category,
+    ageRating,
+    violence,
+    sexAndNudity,
+    alcoholAndDrugs,
+    gambling,
+    explicitLanguage,
+    averageScore,
+    publisher,
+    releaseDate,
+  } = req.body;
+  new Game({
+    title,
+    category,
+    ageRating,
+    violence,
+    sexAndNudity,
+    alcoholAndDrugs,
+    gambling,
+    explicitLanguage,
+    averageScore,
+    publisher,
+    releaseDate,
```

Comment:
Not sure, but, does this work?

`new Game(req.body)`

Reply:
Indeedy, it does. Not sure why I didn't spot that.

Reply:
Awesome, it can be much smaller then!